Add new page for incident mgmt FAQ (#94)
giantswarm/handbook: content/docs/support-and-ops/processes/incident-management.md

---
title: "How to manage an incident FAQ"
owner:
- https://github.com/orgs/giantswarm/teams/team-horizon
confidentiality: public
---

# FAQ

1. What are the P1/P2 severities?

For simplification there are only two severities, critical (P1) or routine (P2). Critical incidents are those which impair a customer production system. Routine incidents are all other regular incidents which don’t impact production and which have a straightforward process.

2. What do the different roles mean? When should they be used? What does setting them do?

> There are two roles, Incident Coordinator who acts as communicator and driver of the incident and Operation Engineer(s) who has the responsibility to investigate the problem. For not critical incidents (P2), Incident Coordinator is not needed and Operation Engineer act as both roles. 

Author: Yeah this should be done by now :)

[More info](https://docs.giantswarm.io/support/p1-process/#roles)

3. What is the correct way to contact a customer if needed? 

> Every incident declared there is a link to the Customer Escalation Matrix which redirect you to intranet customer page.

Reviewer: I don't understand what this means. Where is the link?

Author: BTW I have fixed the naming

4. When is it appropriate to escalate or page another areas on-caller? How do you do that?

> In critical processes (P1), the Incident Coordinator will decide together with Operation Engineer when someone else is needed. In case of routine incident (P2), Operation Engineer should decide when to escalate it based on the impact and knowledge about the problem. To escalate use `/inc escalate` (or the button) in the incident channel and select the team or person that you want to target.

Reviewer: Similar to this - In a P1 incident how does the on-caller pull in an IC? How do they know who to page?

Author: There is a group called `Incident Coordinators` which will manage the routing. Adding it to the explanation now...

> In critical processes (P1), the Incident Coordinator will decide together with Operation Engineer when someone else is needed. In case of routine incident (P2), Operation Engineer should decide when to escalate it based on the impact and knowledge about the problem. To escalate use `/inc escalate` (or the button) in the incident channel and select the team or person that you want to target.

Author: ```suggestion
In critical processes (P1), the Incident Coordinator will decide together with Operation Engineer when someone else is needed. In case of a routine incident (P2), Operation Engineer should decide when to escalate it based on the impact and knowledge about the problem. To escalate use `/inc escalate` (or the button) in the incident channel and select the team or person that you want to target.

__Note__: To call for an incident coordinator you can select the `incident_coordinators` group in the `Who I need?` popup window when you escalate through incident.io.
```

4. How to silence an alert out-of-hours.

> Silences are managed in a [single repo](https://github.com/giantswarm/silences/) and they are a especial Custom Resource. Create a new resource copying from existing and modifying cluster/installation names. If you are in the middle of the night don't hesitate to merge without approval. [More info](https://intranet.giantswarm.io/docs/support-and-ops/processes/silence-management/).

Reviewer 2: ```suggestion
Silences are managed in a [single repo](https://github.com/giantswarm/silences/) and they are a special Custom Resource. Create a new resource copying from existing and modifying cluster/installation names. If you are in the middle of the night don't hesitate to merge without approval. [More info](https://intranet.giantswarm.io/docs/support-and-ops/processes/silence-management/).
```

Reviewer: Unrelated to this document, but that `silence-mangement` page should be updated to include details about the `valid-until` annotation as we shouldn't be creating any without it set to something sensible.

> 5. How should I raise issues I discover during debugging? Do all need issues creating? 

Reviewer 2: I'd add a line saying that all incidents should (ideally) end up in either a new PM, or a comment on an existing PM

Author: done PTAL

4. When is it appropriate to escalate or page another areas on-caller? How do you do that?

In critical processes (P1), the Incident Coordinator will decide together with Operation Engineer when someone else is needed. In case of a routine incident (P2), Operation Engineer should decide when to escalate it based on the impact and knowledge about the problem. To escalate use `/inc escalate` (or the button) in the incident channel and select the team or person that you want to target.

__Note__: To call for an incident coordinator you can select the `incident_coordinators` group in the `Who I need?` popup window when you escalate through incident.io.

4. How to silence an alert out-of-hours.

Silences are managed in a [single repo](https://github.com/giantswarm/silences/) and they are a special Custom Resource. Create a new resource copying from existing and modifying cluster/installation names. If you are in the middle of the night don't hesitate to merge without approval. [More info](https://intranet.giantswarm.io/docs/support-and-ops/processes/silence-management/).

5. How should I raise issues I discover during debugging? Do all need issues creating? 

> Create a [new postmortem](https://github.com/giantswarm/giantswarm/issues/new?assignees=&labels=postmortem%2C+team%2Fnull&template=operations-postmortem.md&title=) that will collect on the main data about the problem and team will take it based on priority.

Author: ```suggestion
Taking into account all incidents should end up with a postmortem, make sure before creating a [new one](https://github.com/giantswarm/giantswarm/issues/new?assignees=&labels=postmortem%2C+team%2Fnull&template=operations-postmortem.md&title=) there is no already an existing one. In the postmortem, you can point to problems found or suggestions raised and the target team will address them based on priority.
```
